src/controllers/dynakube/dynakube_controller_test.go

```diff
@@ -179,7 +179,13 @@ func TestReconcileActiveGate_Reconcile(t *testing.T) {
 		result, err := r.Reconcile(context.TODO(), reconcile.Request{
 			NamespacedName: types.NamespacedName{Namespace: testNamespace, Name: testName},
 		})
+		assert.NoError(t, err)
+		assert.NotNil(t, result)
 
+		// Reconcile twice since proxy secret is created before the stateful set
+		result, err = r.Reconcile(context.TODO(), reconcile.Request{
```

Comment:
you could move this to a separate test and mock the expected before state so you only need to reconcile once and have no other dependencies

Reply:
It's my mistake. There is no need to call Reconcile() twice because proxy is not configured.